### PR TITLE
quotes prevent glob and word splitting + typos

### PR DESCRIPTION
**Pull request by ghost**
shellchecking and fixed misspelling/typos

### CODE REVIEW

File: pulseaudio-control.bash [hunk 1]
```diff
@@ -55,21 +55,21 @@ function getCurNode() {
     local curNodeName
 
     curNodeName=$(pactl info | awk "/Default S${SINK_OR_SOURCE}: / {print \$3}")
-    curNode=$(pactl list s${SINK_OR_SOURCE}s | grep -B 4 -E "Name: $curNodeName\$" | sed -nE "s/^S${SINK_OR_SOURCE} #([0-9]+)$/\1/p")
+    curNode=$(pactl list s"${SINK_OR_SOURCE}"s | grep -B 4 -E "Name: $curNodeName\$" | sed -nE "s/^S${SINK_OR_SOURCE} #([0-9]+)$/\1/p")
```
Comment (marioortizmanero) [inline pulseaudio-control.bash]: How about `"s${SINK_OR_SOURCE}s"`? That looks more natural here. This seems to happen with every change in this PR (other than typos).

**Review by marioortizmanero — COMMENTED** [review]
Many thanks for the contribution!

File: README.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ Options:
         extension.
         Default: none
   --node-nicknames-from <prop>
-        pactl property to use for node names, unless overriden by
+        pactl property to use for node names, unless overridden by
```
Comment (marioortizmanero) [inline README.md:111]: This is actually from the `--help` message in the script. Could you fix it there as well?

Comment (ghost) [reply README.md:111]: My apologies, I am unsure what file or line you are referring to fixing.

**Review by ghost — COMMENTED** [review]
made changes please double check to make sure this is working as intended

File: pulseaudio-control.bash (hunk 1 above)
Comment (ghost) [reply pulseaudio-control.bash]: I've made those changes and I agree that it does look more natural. Please double check to make sure I'm not missing something.

Thank you.

File: README.md (hunk 2 above)
Comment (marioortizmanero) [reply README.md:111]: This one: https://github.com/marioortizmanero/polybar-pulseaudio-control/blob/10b08bcf57f7c7007eebab2d739534b1e1da0c81/pulseaudio-control.bash#L461

**Comment by marioortizmanero** [comment]
Thanks again! Have happy new year :)

CI fails due to unrelated issues, will fix separately.

**Pull request merged**